Backend/dev/migrations/rider-app/1355-change-vp-in-tracking-url.sql

```diff
@@ -0,0 +1,11 @@
+update atlas_app.rider_config SET tracking_short_url_pattern = 'https://nammayatri.in/u?vp={#vp#}&rideId={#rideId#}';
```

Comment:
@ErasedSoul will this be Backward compatible if the Migration is Ran with old code ?

Reply:
**No its not backwards compatible with old code.** 
If we want perfect backwards compatibility we have to do the old way I wrote .. @prakharritik 
**Old code idea:**
Urlpattern -> "https://nammayatri.in/u?rideId="
and we append :   "&vp=shareRide" or "&vp=delivery" in code logic
@khuzema786

Reply:
Wait let me do some changes to make it backward compatible.. discussed with @prakharritik